Doc/reference/compound_stmts.rst

```diff
@@ -312,9 +319,15 @@ when leaving an exception handler::
    >>> print(sys.exc_info())
    (None, None, None)
 
+
 .. index::
    keyword: except_star
 
+.. _except_star:
+
+:keyword:`!except*` clause
+--------------------------
+
 The :keyword:`except*<except_star>` clause(s) are used for handling
 :exc:`ExceptionGroup`\ s. The exception type for matching is interpreted as in
```

Comment:
```suggestion
The :keyword:`except* <except_star>` clause(s) are used for handling
:exc:`ExceptionGroup`\s. The exception type for matching is interpreted as in
```
While you are at it, you could tweak these.

Reply:
I guess so, as long as that doesn't scope-creep too much—I know I have a bad habit of that...